olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -124,6 +124,20 @@ spec:
                     minimum: 1
                     type: integer
                 type: object
+              logging:
+                description: Set logging configuration of the data plane
```

Comment:
```suggestion
                description: Set logging configuration of the data plane (receivers and dispatchers). It is not recommended to use DEBUG or TRACE in production.
```

Reply:
Another option:
```suggestion
                description: Set logging configuration of the data plane (receivers and dispatchers). It is not recommended to use DEBUG or TRACE in production since it's too verbose and degrades performance.
```